README.md

```diff
@@ -158,7 +166,7 @@ Aggregate post-award statistics and individual bid submissions:
 }
 ```
 
-A potential supplier submits a bid for two lots as a single document:
+A potential supplier submits an expression of interest for two lots as a single document:
```

Comment:
I think it's unusual for an expression of interest to only describe the price, so let's keep it as a bid example.

Reply:
I think this example is useful for the case of bids on multiple lots, but if it's to stay a bid (which makes a lot of sense) should we add a third example for an EOI?

Reply:
I think the submissionType is simple enough to not need an example

Reply:
I agree, great I'll leave it as it then